php-practice/section11.php

```diff
@@ -252,15 +279,47 @@ function roll() {
  *
  * Create a Ball object and a BeachBall object.
  * Call roll() on both of them. Then echo out $color for both of them.
- * What happens when you call roll() on the Ball object?
- * What happens with the BeachBall object?
+ * What happens when you call roll() on the Ball object? 
+ * What happens with the BeachBall object? 
+ * ANSWER HERE: 'The ball is rolling'&'The beach ball is rolling, but it\'s slowing quickly'
  *
- * ANSWER HERE:
  */
 
+$ball = new Ball();
+
+$ball->roll();
+
+$beachBall = new BeachBall();
+
+$beachBall->roll();
+
 /**
  * PROBLEM 7
  *
  * Use your imagination to come up with a parent and child class
  * On the child class, override properties and methods from the parent class
  */
+
+class PapaSmurf {
+  public $color = 'Blue, of course';
+
+  function villian() {
+    echo 'Gargamel wants to capture the Smurfs' . PHP_EOL;
+  }
+}
+
+class Smurfette extends PapaSmurf {
+  public $color = 'Yes, we are blue but I am female';
+
+  function villian() {
+    echo 'Gargamel created Smurfette as a trap' . PHP_EOL;
+  }
+}
+
+$papaSmurf = new PapaSmurf();
+
+$papaSmurf->villian();
+
+$smurfette = new Smurfette();
+
+$smurfette->villian();
```

Comment:
We've talked a lot about naming conventions. One foundational principle is that the names of variables/properties should be nouns (e.g. `errorFormatter`, `data`, `users`) while the names of functions/methods should be verbs (e.g. `printOutErrorMessage`, `deleteUser`, `roll`)

With that in mind, what's a better name for the `villian` method that explains more clearly what it does?